src/dashboard/apps/dashboard/settings.py

```diff
@@ -9,18 +8,17 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/5.1/ref/settings/
 """
-import environ
-import os
 
 from pathlib import Path
 
+import environ  # type: ignore
```

Comment:
You should better ignore it at a configuration level instead of the code (_i.e._ in the `pyproject.toml` file).

```suggestion
import environ
```

Reply:
✅